tests/e2e/conftest.py

```
@@ -0,0 +1,14 @@
+"""Pytest configuration for e2e tests."""
+
+import pytest
+
+# E2E-specific fixtures can be defined here
+
+# Fixtures from test_cli/conftest.py
+import pytest
```

Ruff (0.13.3)

Redefinition of unused `pytest` from line 3

Remove definition: `pytest`

(F811)